vnext/package.json

```diff
@@ -69,5 +69,56 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
```

Comment:
Does anything live in `etc` or `flow` that we need? I was under the impression we didn't put anything in `flow` anymore, but I wonder if we have some build steps laying around still.

Reply:
I'll leave that for others, I don't know yet :)

Reply:
`/flow` has:
```
console.js
fbjs.js
jest.js
Position.js
Promise.js
Stringish.js
use-subscription.js
```

`/etc` has:
```
react-native-windows.api.md
```
the file has a comment:
> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).

Reply:
You can remove `etc` since anyone with the package will already have TS definitions. API extractor docs are just for viewing in the repo (though they're not super useful right now).

Thanks for checking on `flow`. Got confused because we used to also have `flow-typed` with flow definitions that was removed upstream. FB exposes the directory in their package, so we will want to keep doing the same.

Reply:
/etc can certainly be excluded.
Maybe just open an issue to look into if we really need the flow files.

Reply:
#4809 & #4810